deploy/deploy_analytics_terminal.yml

```diff
@@ -21,6 +21,7 @@
     - redis
     - maven
     - intellij
+    - neo4j
```

Comment:
@olivermel We are not actually installing this on the Analytics Terminal or the App Servers, but rather setting up a new server just for Neo4J.  So you need to create a new entry in the Vagrantfile, and a new deploy_neo4j.yml file for deploying this server.  The playbook should include the usual "common" and "iptables" rules, plus whatever dependencies exist for neo4j (at least the JDK role).